src/time.rs

```diff
@@ -1,5 +1,6 @@
 //! Time units
 
+/*
```

Comment:
simply delete it instead of commenting it out. If we feel nostalgic, we can always go back and look at it - that's the beauty of git ;)